### DISCUSSION TITLE
Dashboard View Icon Problem

### DISCUSSION THREAD

**Discussion by rodolfomachado** [elax46/custom-brand-icons#497] [Q&A] [answered]
Hi there,

I'm trying to use roborock icon in one of my view but it's not working in my mobile. 
![image](https://github.com/elax46/custom-brand-icons/assets/42542786/35a47021-a361-457e-b8bc-0260bb2f41b6)

When I access my HA via laptop it works fine.
![image](https://github.com/elax46/custom-brand-icons/assets/42542786/66b99a6c-7164-40e8-8ceb-df888f2ca373)

I installed custom-brand-icons through HACS and I'm also added the following lines to configuration.yaml
![image](https://github.com/elax46/custom-brand-icons/assets/42542786/6bab1197-eb2d-43ae-9d5d-433fa8a0d1de)

Here is my Resource:
![image](https://github.com/elax46/custom-brand-icons/assets/42542786/d7dba081-4e1c-4d33-9bd0-88d4f830315c)

What am I missing?

**Comment by elax46** [comment, marked as answer]
Unfortunately it is a cache problem that depends on the application, I also have the same problem in my instance
